Applications/MaterialManager/Contracts/Material/Boards/BoardType.cs

```diff
@@ -149,6 +149,12 @@ public class BoardType : IExtensibleDataObject, IContainsUnitSystemDependentProp
         [JsonProperty(Order = 33)]
         public string? ArticleNumber { get; set; }
 
+        /// <summary>
+        /// Gets or sets the gtin.
+        /// </summary>
+        [JsonProperty(Order = 33)]
```

Comment:
34

Reply:
34 was taken, I made it 36. Strangely the EdgebandType does not have this Order at all. Resolved.

Reply:
The order number is just for displaying the Json in an explicit order when accessing it through and HTTP Request so it is only for elegance but not really relevant

Reply:
Shouldn't we add it for Edgebands?